src/main/resources/typescript-axios-webpack-module-federation/api.mustache

```diff
@@ -0,0 +1,13 @@
+/* tslint:disable */
+/* eslint-disable */
+
+import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/dist/utils';
+import type { AxiosInstance } from 'axios';
+import {
+{{>endpointList}}  } from './index';
+
+export const {{{clientName}}} = (BASE_PATH: string, instance: AxiosInstance) => {
```

Comment:
The `instance` should be optional. It should be of type `ApiConfig` not `AxiosInstance`